bin/clover/src/pipelines/azure/spec.ts

```diff
@@ -125,15 +134,23 @@ export function parseAzureSpec(
 ///
 /// Recursively normalizes child properties as well.
 export function normalizeAzureSchema(
-  prop: JSONSchema | undefined,
-  processing: JSONSchema[],
+  prop: AzureSchemaDefinition | undefined,
+  processing: AzureSchemaDefinition[],
+  definitions?: AzureDefinitions,
+  discriminatorCollector?: Record<string, Record<string, string>>,
 ): NormalizedAzureSchema | undefined {
   if (prop === undefined) return undefined;
   if (processing.includes(prop)) return undefined;
   processing.push(prop);
   try {
     // Flatten the schema, merging allOf props and such, before we normalize type/format
-    const normalized = flattenAzureSchema(prop, {}, processing);
+    const normalized = flattenAzureSchema(
+      prop,
+      {},
+      processing,
```

Comment:
oy, at some point we need to make the flattener an object with all this crap in it :) (Not right now)